TanukiKitTests/ProjectTests.swift

```diff
@@ -6,7 +6,7 @@ class ProjectTests: XCTestCase {
     // MARK: Actual Request tests
 
     func testGetProjectsCOM() {
-        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?access_token=12345&page=1&per_page=100", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
+        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?access_token=12345&archived=&order_by=created_at&page=1&per_page=20&search=&simple=&sort=desc&visibility=", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
```

Comment:
I tested all the requests manually, they all work with empty parameters, the problem is we support so many parameters, if I were to do an if then else for each possible combination of used parameters (like in OctoKit) it would take me exactly 3,628,800 if statements.